src/vs/workbench/parts/explorers/browser/treeExplorerViewletService.ts

```diff
@@ -0,0 +1,76 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
@octref I find the naming of this service weird and hard to understand what it does. can we maybe rename it to something that better tells us what the purpose is? I see the name is `ITreeExplorerViewletService` but it gets contributed as `customViewletService` which seems weird. Mayb `IExtensionTreeService`? Afaik it seems only tree related but not viewlet related.